lib/Parser/Parser.cpp

```diff
@@ -937,6 +938,12 @@ bool Parser::parseLine(std::string &ErrStr) {
 		  return false;
 		}
                 PowOfTwo = true;
+	      } else if (CurTok.PatternString[i] == '-') {
+		if (Negative) {
+		  ErrStr = makeErrStr(TP, "repeated '-' flag");
+		  return false;
+		}
+                Negative = true;
```

Comment:
Fixed. The tabs were sitting in Parser.cpp in previous commits too. Hope it looks fine now.